.bluemix/wizard.yml

```diff
@@ -89,5 +89,7 @@ form:
       $ref: wizard_optional_tools.json
 
   wizard_summary:
+    parameters:
+      test: true
```

Comment:
Can you remind me of the purpose of `test:true` ?

Reply:
Temporary workaround for null pointer in otc-ui. Once that issue is fixed, we can ditch it

Reply:
Fix is in this PR: https://github.ibm.com/org-ids/otc-ui/pull/2377